### ISSUE TITLE
Replacer Wear and Tear 

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#327]
chuche952
replacer:replacer is supposed to wear out on use, but it doesn't on Pandora. To offset this, there is replacer:replacer_technic, which is chargable - but because wear is deactivated on Pandora, it makes no use. It makes sense to activate wear for both tools.

SX
Wear does not work well with chargeable tools, that's because wear value is used to display remaining charge. Both can be done but it requires some workarounds to display both. Maybe for basic replacer tool but not sure, definitely this should be possible with configuration but not sure if it is good here. Might be, basic replacer is relatively cheap and repair is easy here so could possibly try.

The4spaceconstants 
wear and charge use same value

SX 
They do after my rewrite but does not do that on original Technic or most other forks, those use separate metadata value for actual charge and copy that to wear value to display remaining charge on item slot visual. Reason for that is inaccurate wear with certain charge/discharge ratios because wear is 16bit while metadata values can be whatever you like.

**Comment by OgelGames** [comment]
I thought the replacer didn't use any wear or charge when only replacing a single node, but did when using field or crust modes?

Maybe it should be dependent on whether you would need a tool to dig it normally, similar to how node breakers work.

https://github.com/mt-mods/pipeworks/blob/ebe23316193e563a264b0f6fec017c02bbc46d4c/wielder.lua#L34-L57

**Comment by SwissalpS** [comment]
> I thought the replacer didn't use any wear or charge when only replacing a single node, but did when using field or crust modes?

yes that's how it is. I understand chuche would like it to wear out (English is not their first language)
